jerry-core/ecma/builtin-objects/ecma-builtin-helpers-date.c

```diff
@@ -333,7 +333,7 @@ ecma_date_daylight_saving_ta (jerry_time_zone_t *tz, /**< time zone information
    * TODO: Fix daylight saving calculation.
    *       https://github.com/jerryscript-project/jerryscript/issues/1661
    */
-  return tz->daylight_saving_time * ECMA_DATE_MS_PER_HOUR;
+  return (ecma_number_t) tz->daylight_saving_time * ECMA_DATE_MS_PER_HOUR;
```

Comment:
My point is:

- There is not much point to use a date which works dates only between 1960-1980.
- People will be unaware of this fact.

So it would be better to add a # error that date does not support float32 at the moment. If we would rewrite date to use uint64 that would fix the problem.